index.html

```diff
@@ -24,7 +24,7 @@
     -->
     <base href="/">
 
-    <link rel="icon" href="images/favicon.ico">
+    <link rel="icon" href="images/favicon.png">
```

Comment:
Apart from the icon in the index.html, you need to modify the images referenced in the manifest.json file so as all icons are coherent in all devices supporting PWA